# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/WebrtcTCPSocketChild.h
# Commit: b879a0044cf3
# Full Hash: b879a0044cf39f0230879d075c915485affb7707
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Teach WebrtcTCPSocket to bind to a specific host/port. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45102
# ==============================================================================

diff -r f5353ea9f1df -r b879a0044cf3 media/mtransport/ipc/WebrtcTCPSocketChild.h
--- a/media/mtransport/ipc/WebrtcTCPSocketChild.h	Tue Sep 17 18:17:14 2019 +0000
+++ b/media/mtransport/ipc/WebrtcTCPSocketChild.h	Tue Sep 17 18:17:18 2019 +0000
@@ -30,6 +30,7 @@
   explicit WebrtcTCPSocketChild(WebrtcTCPSocketCallback* aProxyCallbacks);
 
   void AsyncOpen(const nsCString& aHost, const int& aPort,
+                 const nsCString& aLocalAddress, const int& aLocalPort,
                  const net::LoadInfoArgs& aArgs, const nsCString& aAlpn,
                  const dom::TabId& aTabId,
                  NrSocketProxyConfig::ProxyPolicy aProxyPolicy);